Library/Formula/json-fortran.rb

```diff
@@ -0,0 +1,73 @@
+  sha256 "97f258d28536035ef70e9ead5c7053e654106760a12db2cc652587ed61b76124"
+
+  head "https://github.com/jacobwilliams/json-fortran.git"
+
+  option "with-unicode-support", "Build json-fortran to support unicode text in json objects and files"
+  option "with-keg-only", "Keg only install (Skip linking json-fortran.) Required to support multiple Fortran compilers."
+  option "without-make-test", "Skip running build-time tests (not recommended)"
+
+  depends_on "cmake" => :build
+  depends_on "cmake"
+  depends_on "robodoc" => :recommended
+  depends_on :fortran
+
+  keg_only "User selected keg only install" if build.with? "keg-only"
+
+  def caveats; <<-EOS.undent
+      json-fortran must be compiled with a relatively new Fortran compiler.
+      gfortran >= 4.9.2 will work, which is what is installed with the Home-
+      brew gcc package on OS X Yosemite. Intel's Fortran compiler, ifort >=
+      13.1, will also work, and, reportedly, so will NAG >= 6.0; HOWEVER, it
+      is only recommended that json-fortran be installed with Homebrew using
+      gfortran.  This is the default so long as the `FC` environment variable
+      is not set.  If you plan on supporting multiple Fortran compilers, then
+      you probably should install json-fortran by hand using the CMake build
+      script included in the project. If you wish to support multiple Fortran
+      compilers AND install a copy of json-fortran with Homebrew, we suggest
+      you do so with the `--keg-only` option which will prevent Homebrew cre-
+      ating symbolic links in /usr/local/lib and /usr/local/include.
+      EOS
+  end
+
+  def install
+    mktemp do
+      args = []
+      args << "-DUSE_GNU_INSTALL_CONVENTION:BOOL=TRUE" if build.without? "keg-only"
+      args << "-DENABLE_UNICODE:BOOL=TRUE" if build.with? "unicode-support"
+      args << "-DSKIP_DOC_GEN:BOOL=TRUE" if build.without? "robodoc"
+      system "cmake", buildpath, *(args + std_cmake_args)
+      system "make"
+      system "make", "test" if build.with? "make-test"
+      system "make", "install"
+    end
+  end
+
+  test do
+    (testpath/"CMakeLists.txt").write <<-EOS.undent
+      cmake_minimum_required ( VERSION 2.8.8 FATAL_ERROR )
+      enable_language ( Fortran )
+      project ( jf_test NONE )
+      find_package ( jsonfortran-${CMAKE_Fortran_COMPILER_ID} 4.1.0 REQUIRED )
+      include_directories ( "${jsonfortran_INCLUDE_DIRS}" )
+      file ( WRITE json_test.f90 "program json_test
+          use json_module
+          use ,intrinsic :: iso_fortran_env ,only: error_unit
+          implicit none
+          call json_initialize()
+          if ( json_failed() ) then
+            call json_print_error_message(error_unit)
+            stop 2
+          endif
+        end program\n")
+      add_executable ( json_test json_test.f90 )
+      target_link_libraries ( json_test jsonfortran )
+    EOS
+    system "cmake", ".", "-DCMAKE_Fortran_COMPILER=gfortran", *(std_cmake_args)
```

Comment:
with only having `depends_on cmake => :build` the `brew test json-fortran` fails when run by testbot under the Jenkins CI setup